docs/reverse_proxy.rst

```diff
@@ -85,6 +85,28 @@ Let's assume that we expect clients to connect to our server at
           </Location>
       </VirtualHost>
 
+* HAProxy::
+
+      frontend https
+        bind 0.0.0.0:443 v4v6 ssl crt /etc/ssl/haproxy/ strict-sni alpn h2,http/1.1
+        bind :::443 ssl crt /etc/ssl/haproxy/ strict-sni alpn h2,http/1.1
+        
+        # A standard website
```

Comment:
let's get rid of this? we don't have it for any of the other proxies and I think it's confusing.

Reply:
All right.